api/v1beta1/hyperconverged_types.go

```diff
@@ -244,6 +249,11 @@ type HyperConvergedFeatureGates struct {
 	// +optional
 	// +kubebuilder:default=true
 	EnableCommonBootImageImport bool `json:"enableCommonBootImageImport"`
+
+	// deploy resources (kubevirt tekton tasks and example pipelines) in Tekton tasks operator
+	// +optional
+	// +kubebuilder:default=false
```

Comment:
due to same limitation on the defaulting mechanism in kubebuilder, you have to explicitly mention the default for DeployTektonTaskResources also on lines 53 and 498